Packs/ThreatConnect/Integrations/ThreatConnectV3/ThreatConnectV3.py

```diff
@@ -318,6 +318,9 @@ def tc_get_indicators_command(client: Client, args: dict, confidence_threshold:
     indicator_associations = args.get('indicator_associations', indicator_associations)
     group_associations = args.get('group_associations', group_associations)
     tql_prefix = ''
+    if summary:
+        summary = f' AND summary EQ "{summary}"'
```

Comment:
what if the summary value contains `"`?

Reply:
Because the tql needs it as a string

Reply:
Need to escape the `"`  char in the summary variable. otherwise, it will break the TQL syntax.